nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/test/groovy/org/apache/nifi/reporting/script/ScriptedReportingTaskGroovyTest.groovy

```diff
@@ -83,7 +83,8 @@ class ScriptedReportingTaskGroovyTest {
     @Test
     void testProvenanceGroovyScript() {
         def uuid = "10000000-0000-0000-0000-000000000000"
-        def attributes = ['abc': 'xyz', 'xyz': 'abc', 'filename': "file-$uuid", 'uuid': uuid]
+        // FIXME java11 removed usage of GString, causes issue in java code due to not being able to cast GString to String
```

Comment:
Same comment here about restoring until Java 11 support is in place

Reply:
I've reverted this change, since it's out of scope with changes for the Metrics module.  In an upcoming PR for the Groovy 2.5.x update, this change will be brought in.  Good catch!